grafana-plugin/src/models/filters/filters.helpers.ts

```diff
@@ -7,3 +11,13 @@ export const getApiPathByPage = (page: string) => {
     }[page] || page
   );
 };
+
+export const normalizeFilters = (filters: FiltersValues, filterOptions: FilterOption[]) => {
```

Comment:
We don't really normalize the object here, we just conditionally map one of its values. How about the name: "convertRelativeToAbsoluteDateForDateRange"?

Reply:
I think it should have more generic name because the main idea is to cast internal representation in a format that the backend understands, maybe `castFiltersToBackendFormat`?

Reply:
Sounds good!